live-rates/README.md

```diff
@@ -0,0 +1,28 @@
+# Chainlink External Adapter for Live-Rates
+
+## Input Params
+
+- `base`, `from`, `symbol`, or `rate`: The symbol of the currency to query
```

Comment:
Since this is a required parameter we should mark it as such.

We have started using a table README to make things such as this more clear, you can see an example here:
https://github.com/smartcontractkit/external-adapters-js/blob/develop/alphavantage/README.md